src/types/ErrorType.ts

```diff
@@ -0,0 +1,12 @@
+export enum ErrorType {
+  GetError = 'Unable to get response from the server',
+  PostError = 'Unable to add a todo',
+  DeleteError = 'Unable to delete a todo',
+  PatchError = 'Unable to update a todo',
+  success = 'success',
+  get = "get"
```

Comment:
```suggestion
  PatchError = 'Unable to update a todo',
  Success = 'success',
  Get = "get"
```
All enum properties should be capitalized